inlong-sort/pom.xml

```diff
@@ -40,7 +40,7 @@
         <module>sort-connectors</module>
         <module>sort-core</module>
         <module>sort-dist</module>
-        <module>sort-end-to-end-tests</module>
+        <!--<module>sort-end-to-end-tests</module>-->
```

Comment:
Please do not change this module, the unit tests error for this module will be fixed later.

Reply:
sorry, foget to uncomment it.